src/core/ledger/grainAllocation.js

```diff
@@ -66,17 +66,20 @@ export type AllocationPolicy =
 export type BalancedPolicy = {|
   +policyType: Balanced,
   +budget: G.Grain,
+  +mapping?: Function,
 |};
 
 export type ImmediatePolicy = {|
   +policyType: Immediate,
   +budget: G.Grain,
+  +mapping?: Function,
```

Comment:
The name "mapping" isn't 100% clear, leads me to believe that this is a mapping for grain amounts, but seems like its actually a `credMapping`